### PR TITLE
Added tests for `mesheryctl/pkg/utils/auth.go`

### PR DESCRIPTION
**Pull request by shikharvashistha**
Signed-off-by: Shikhar Vashistha <>

**Description**

This PR is in accordance to issue #3988 where I've added some  more tests for `mesheryctl/pkg/utils/auth.go` 
**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by piyushsingariya** [comment]
> @piyushsingariya Could you check this PR?

will review this thoroughly soon.

**Comment by navendu-pottekkat** [comment]
@shikharvashistha Could you check the comments made in the review and address those?

**Comment by shikharvashistha** [comment]
> @shikharvashistha Could you check the comments made in the review and address those?

Yeah

**Comment by shikharvashistha** [comment]
> 
> 
> Checking in... it has been awhile since we&#39;ve heard from you on this issue. Are you still working on it? Please let us know and please don&#39;t hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
> 
> ---
> 
>  > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](http://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile: 

Yes I'm

File: mesheryctl/pkg/utils/utils_test.go [hunk 1]
```diff
@@ -363,3 +367,139 @@ spec:
...
+			},
+			filepath: " ",
+			wantErr:  false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.filepath, func(t *testing.T) {
+			if err := AddAuthDetails(tt.request, tt.filepath); (err != nil) == tt.wantErr {
+				t.Errorf("AddAuthDetails() error = %v, wantErr %v", err, tt.wantErr)
+			}
```
[the remark this answers is not in the document]
Comment (shikharvashistha) [reply mesheryctl/pkg/utils/utils_test.go:400]: @lee

**Pull request closed without merging**